src/__tests__/componentHelpers/FilterYears.tsx

```diff
@@ -39,7 +40,7 @@ export const FilterYearsTests = (
       });
       fireEvent.change(fromInput, { target: { value: 1990 } });
       expect(fromInput).toHaveValue(1990);
-      userEvent.click(applyButton);
+      await user.click(applyButton);
```

Comment:
It looks like there are two different user event actions we're using across all of these tests:

1. `fireEvent`
3. `userEvent`

It looks like rtl recommends using `userEvent`. Let's aim to replace `fireEvent` with `userEvent`, either in this PR or a follow-up one.